Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -247,18 +246,18 @@ void ConfigurePluginsServices()
             {
                 _logger.Debug("Configuring services in plugins:");
                 
-                foreach (var type in TypeHelper.GetPluginTypes())
+                foreach (var servicesConfigurationActivity in TypeHelper.GetAssemblyTypes<IServicesConfigurationActivity>())
                 {
-                    _logger.Debug($"Plugin {type.Name}");
+                    _logger.Debug($"Plugin {servicesConfigurationActivity.Name}");
```

Comment:
Debug log entry should probably indicate this is a "Services configuration" rather than a "Plugin"...

```csharp
_logger.Debug($"Executing services configuration activity '{servicesConfigurationActivity.Name}'...");
```